### ISSUE TITLE
Bindable FlexLayout not sized and visible on Windows

### ISSUE THREAD

**Issue by davidortinau** [dotnet/maui#3475]
### Description

FlexLayout on Android

![image](https://user-images.githubusercontent.com/41873/142696793-4fecc488-7782-4da0-99a7-f29a84efdf1d.png)

FlexLayout on Windows

<img width="581" alt="image" src="https://user-images.githubusercontent.com/41873/142696593-9cae8d55-4f03-473a-8f47-2c539140511d.png">

StackLayout on Windows

<img width="599" alt="image" src="https://user-images.githubusercontent.com/41873/142696628-3472f160-041a-4636-8fdc-0133b81afeca.png">

Code

```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:models="clr-namespace:ControlGallery.Models"
             x:Class="ControlGallery.Pages.ColorsPage"
             Title="Colors">
    <ContentPage.Content>
        <ScrollView>
            <StackLayout Padding="{OnPlatform iOS='30,60,30,30', Android='15', Default='30'}">
                <Label Text="Colors" StyleClass="Headline"/>
                <BoxView StyleClass="HRule"/>
                
                <FlexLayout Wrap="Wrap" 
                            AlignContent="SpaceEvenly" 
                            Margin="2,15">
                    <BindableLayout.ItemsSource>
                        <x:Array Type="{x:Type models:ColorBrush}">
                            <models:ColorBrush Brush="{StaticResource PrimaryBrush}" BrushName="Primary"/>
                            <models:ColorBrush Brush="{StaticResource SecondaryBrush}" BrushName="Secondary"/>
                            <models:ColorBrush Brush="{StaticResource TertiaryBrush}" BrushName="Tertiary"/>
                            <models:ColorBrush Brush="{StaticResource NeutralBrush}" BrushName="Neutral"/>
                            <models:ColorBrush Brush="{StaticResource NeutralLighterBrush}" BrushName="NeutralLighter"/>
                            <models:ColorBrush Brush="{StaticResource NeutralDarkerBrush}" BrushName="NeutralDarker"/>
                            <models:ColorBrush Brush="{StaticResource BlackBrush}" BrushName="Black"/>
                            <models:ColorBrush Brush="{StaticResource WhiteBrush}" BrushName="White"/>
                        </x:Array>
                    </BindableLayout.ItemsSource>
                    <BindableLayout.ItemTemplate>
                        <DataTemplate>
                            <StackLayout WidthRequest="160" HeightRequest="160" Margin="8,0,8,8">
                                <Ellipse WidthRequest="100" HeightRequest="100"
                                         HorizontalOptions="Center"
                                         Fill="{Binding Brush}"/>
                                <Label Text="{Binding BrushName}"/>
                                <Label Text="{Binding BrushValue}"/>
                            </StackLayout>
                        </DataTemplate>
                    </BindableLayout.ItemTemplate>
                </FlexLayout>
            </StackLayout>
        </ScrollView>
    </ContentPage.Content>
</ContentPage>
```

Source: https://github.com/davidortinau/ControlGallery

### Steps to Reproduce

Run the Colors controls page sample

### Version with bug

Preview 11

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Win11

**Comment by Amir-Care** [comment]
Repro on VS 17.2.0 Preview 2.0 [32208.484.main]

**Comment by ggutschi** [comment]
Is there any known workaround for this? This is currently a showstopper for me :(

**Comment by davidortinau** [comment]
The FlexLayout has no size. If I give it size, then the content appears. 

<img width="840" alt="image" src="https://user-images.githubusercontent.com/41873/164578457-fb4959da-d5cc-4610-951e-6a3465ea32a7.png">

```
<FlexLayout Wrap="Wrap" Background="Red"
                            AlignContent="SpaceEvenly" 
                            HorizontalOptions="Fill"
                            VerticalOptions="Fill"
                            HeightRequest="400"
                            WidthRequest="400"
                            Margin="2,15">
```

Using Minimum width/height seems to work fairly well, though as I resize the window the content will get clipped by the layout at times.

<img width="758" alt="image" src="https://user-images.githubusercontent.com/41873/164578731-a48df052-19aa-40c2-88f3-7af93ff3dab1.png">

<img width="516" alt="image" src="https://user-images.githubusercontent.com/41873/164578806-8cdb8874-4681-45e1-a679-a9819f037eac.png">

**Comment by hartez** [comment]
Fixed by #6085
